app/views/admin/attachments/edit.html.erb

```diff
@@ -1,11 +1,16 @@
 <% page_title "Editing #{attachment.attachable_model_name} #{attachment.readable_type} attachment" %>
```

Comment:
You probably want to follow this pattern:

```
<% content_for :page_title, "Editing #{attachment.attachable_model_name} #{attachment.readable_type} attachment" %>
<% content_for :title, "Edit #{attachment.readable_type} attachment" %>
<% content_for :context, "Attachments for #{attachment.attachable_model_name}" %>
```

Reply:
👍